### DISCUSSION TITLE
Spacing changes between light and dark mode

### DISCUSSION THREAD

**Discussion by wangenau** [pradyunsg/furo#316] [Potential Issue] [answered]
Hi,

while playing around with your beautiful theme I encountered that the spacing changes between light and dark mode in code blocks.
This also happens for the same pygments style with same fonts.
I have attached two screenshots of the same screen selection to illustrate the difference.

<img src="https://user-images.githubusercontent.com/15988797/143264934-54187d64-3203-4a3d-9ab1-492e81d1beab.png" alt="light" width="250"/>
<img src="https://user-images.githubusercontent.com/15988797/143264961-8abd8ecd-8939-47ef-b795-f32c48245a4d.png" alt="dark" width="250"/>

**Comment by pradyunsg** [comment, marked as answer]
Fixed in 0c4018fa730d12dc2144745b75c33b922b7cf7d4. 😅 

I'll cut a release over the weekend.

**Reply by wangenau** [reply]
Awesome, thanks a lot.
